plugin/src/main/groovy/com/kezong/fataar/ResourceModifier.groovy

```diff
@@ -102,6 +114,69 @@ class ResourceModifier {
         Set<String> formats = new HashSet<>()
         Element enumDefinition
         int occurrences = 0
+        Element firstDefinition = null
+        List<Element> declareStyleableElements = new ArrayList<>()
```

Comment:
enumDefinition and firstDefinition are redundant, since they can be found in  declareStyleableElements.